plugins/modules/ini_file.py

```diff
@@ -257,6 +263,14 @@
     value: xxxxxxxxxxxxxxxxxxxx
     mode: '0600'
     state: present
+- name: Update the option and indent with spaces
+  community.general.ini_file:
+    path: /etc/influxdb/config.toml
```

Comment:
As the file extension indicates this is NOT an INI file, but a [TOML file](https://en.wikipedia.org/wiki/TOML).

Using `ini_file` for a TOML file is generally not a good idea, and definitely should not appear in the examples to avoid suggesting this to users.